3_callbacks/README.md

```diff
@@ -0,0 +1,43 @@
+```c++
+#include <nan.h>
+
+NAN_METHOD(RunCallback) {
+  v8::Local<v8::Function> cb = info[0].As<v8::Function>();
+  const unsigned argc = 1;
+  v8::Local<v8::Value> argv[argc] = { Nan::New("hello world").ToLocalChecked() };
+  Nan::MakeCallback(Nan::GetCurrentContext()->Global(), cb, argc, argv);
```

Comment:
This should use [`Nan::Call`](https://github.com/nodejs/nan/blob/master/doc/maybe_types.md#api_nan_call). `Nan::MakeCallback` should only be used when originating from libuv. They will behave the same in most cases, but there are some fine details in certain corner cases that can lead to unwanted behavior.

Reply:
I was just documenting what's already there in the example, which I didn't modify in this case. https://github.com/nodejs/node-addon-examples/blob/master/3_callbacks/nan/addon.cc

I think we should merge this as-is and file a separate issue that the example (and then its documentation) should be updated.